src/pretix/control/views/vouchers.py

```diff
@@ -225,6 +226,23 @@ def post(self, request, *args, **kwargs):
             return super().post(request, *args, **kwargs)
 
 
+class VoucherGo(EventPermissionRequiredMixin, View):
+    permission = 'can_view_vouchers'
+
+    def get_voucher(self, code):
+        return Voucher.objects.get(code=code, event=self.request.event)
+
+    def get(self, request, *args, **kwargs):
+        code = request.GET.get("code", "").upper().strip()
```

Comment:
Instead of calling ``upper()`` here, please remove this and use ``Voucher.objects.get(code__iexact=code, …)`` above, this is more failsave as vouchers aren't strongly guaranteed to be uppercase.